exporter/awsemfexporter/metric_translator.go

```diff
@@ -61,6 +62,15 @@ type CWMetrics struct {
 	Fields       map[string]interface{}
 }
 
+// GroupedCWMetric defines 
+type GroupedCWMetric struct {
+	Namespace    string
+	Metrics      map[string]interface{}
+	Timestamp    int64
+	Dimensions   map[string]interface{}
+	MetricUnits  map[string]string
```

Comment:
how do we make sure the metrics match the right dimensions? we have two separate maps for metrics and dimensions.

Reply:
fixed